Kernel/Devices/USB/UHCIController.h

```diff
@@ -58,6 +62,11 @@ class UHCIController final : public PCI::Device {
 
     void create_structures();
     void setup_schedule();
+    size_t poll_transfer_queue(QueueHead* transfer_queue);
```

Comment:
Is it valid to call this with `transfer_queue: nullptr`? Otherwise let's take a `QueueHead&`